### PR TITLE
Fix mvn path from IT on Windows

### PR DESCRIPTION
**Pull request by scottkurz**

### CODE REVIEW

**Comment by scottkurz** [comment]
Seems from articles like this:
https://www.mkyong.com/java/how-to-detect-os-in-java-systemgetpropertyosname/

that this may be a specific-enough  test

**Comment by scottkurz** [comment]
This works if `mvn` is in the PATH but I'd prefer to avoid worrying about my PATH.

File: boost-maven/boost-maven-plugin/src/it/test-run-server/src/test/java/it/LibertyRunMojoIT.java [hunk 1]
```diff
@@ -22,30 +22,46 @@
...
 
+    private static String mvnCmd;
+
+    public static void setupMvnPath() {
+
+        String mvnPath = System.getenv("M2_HOME") + "/bin/mvn";
+
+        String osName = System.getProperty("os.name").toLowerCase();
+
+        if ((osName.indexOf("win") >= 0)) {
```
Comment (chyt) [inline boost-maven/boost-maven-plugin/src/it/test-run-server/src/test/java/it/LibertyRunMojoIT.java]: Here, please use the OSUtil method to check for Windows: https://github.com/WASdev/ci.common/blob/master/src/main/java/net/wasdev/wlp/common/plugins/util/OSUtil.java

File: boost-maven/boost-maven-plugin/src/it/test-run-server/src/test/java/it/LibertyRunMojoIT.java [hunk 2]
```diff
@@ -22,30 +22,46 @@
...
+            mvnCmd = "cmd.exe /c " + mvnPath;
+        } else {
+            mvnCmd = mvnPath;
+        }
+    }
+
     @BeforeClass
     public static void init() throws Exception {
-        URL = "http://localhost:8080/";
```
Comment (chyt) [inline boost-maven/boost-maven-plugin/src/it/test-run-server/src/test/java/it/LibertyRunMojoIT.java]: This is just cleanup of an unused variable right?

Comment (chyt) [reply boost-maven/boost-maven-plugin/src/it/test-run-server/src/test/java/it/LibertyRunMojoIT.java]: Never mind, saw you moved it to the class level.

**Pull request merged**